apps/ledger-live-mobile/e2e/page/wallet/portfolio.page.ts

```diff
@@ -56,39 +65,39 @@ export default class PortfolioPage {
 
   async navigateToStakeFromTransferMenu() {
     await scrollToId(this.stakeMenuButtonId, this.transferScrollListId);
-    return tapByElement(this.stakeTransferMenuButton());
+    await tapByElement(this.stakeTransferMenuButton());
   }
 
-  navigateToReceiveFromTransferMenu() {
-    return tapByElement(this.receiveTransfertMenuButton());
+  async navigateToReceiveFromTransferMenu() {
+    await tapByElement(this.receiveTransfertMenuButton());
   }
 
   async receive() {
-    const element = getElementById("receive-button");
-    await element.tap();
+    await tapById(this.receiveButton);
   }
 
-  async waitForPortfolioReadOnly() {
+  async expectPortfolioReadOnly() {
+    await expect(this.portfolioSettingsButton()).toBeVisible();
     await waitForElementById(this.readOnlyItemsId);
-    expect(await getTextOfElement(this.graphCardBalanceId)).toBe(this.zeroBalance);
+    jestExpect(await getTextOfElement(this.graphCardBalanceId)).toBe(this.zeroBalance);
     for (let index = 0; index < 4; index++)
-      expect(await getTextOfElement(this.assetBalanceId, index)).toBe(this.zeroBalance);
+      jestExpect(await getTextOfElement(this.assetBalanceId, index)).toBe(this.zeroBalance);
   }
 
-  openViaDeeplink() {
-    return openDeeplink(baseLink);
+  async openViaDeeplink() {
+    await openDeeplink(baseLink);
   }
 
-  openWalletTabMarket() {
-    return tapByElement(this.walletTabMarket());
+  async openWalletTabMarket() {
+    await tapByElement(this.walletTabMarket());
   }
 
-  openMyLedger() {
-    return tapByElement(getElementById("TabBarManager"));
+  async openMyLedger() {
+    await tapById(this.managerTabBarId);
   }
 
-  openEarnApp() {
-    return tapByElement(this.earnButton());
+  async openEarnApp() {
```

Comment:
same remark

Reply:
It's part of a path that should be tested.